test/libsolidity/ViewPureChecker.cpp

```diff
@@ -53,6 +53,8 @@ BOOST_AUTO_TEST_CASE(environment_access)
 	};
 	if (solidity::test::CommonOptions::get().evmVersion().hasStaticCall())
 		view.emplace_back("address(0x4242).staticcall(\"\")");
+	if (solidity::test::CommonOptions::get().evmVersion().hasBlobHash())
+		view.emplace_back("blobhash(7)");
```

Comment:
I wanted to suggest using the new predicate here too, but actually these tests look like they should just be extracted into `syntaxTests/viewPureChecker/` since all they're checking are warnings and errors.

We should do that after we merge this.

Reply:
I mean, it's **very** low priority. But it's also very easy to do and updating this test is an annoyance that has come up in multiple reviews already, so we should do it in a spare moment.